### PR TITLE
Viafoura Trace Added

### PR DESCRIPTION
**Pull request by dhavalveera**
### Technology Request

**Examples:**

https://www.[stwnewspress.com](https://www.stwnewspress.com/)/
https://www.vaildaily.com/
https://www.sportsnet.ca/
https://www.[salemnews.com](https://www.salemnews.com/)/
https://www.sgvtribune.com/

### CODE REVIEW

File: src/technologies/v.json [hunk 1]
```diff
@@ -410,6 +410,23 @@
     "scriptSrc": "\\.viabill\\.com/",
     "website": "https://viabill.com"
   },
+  "Viafoura": {
+    "cats": [
+      86
+    ],
+    "description": "Viafoura is an audience engagement and social monetization platform.",
+    "dom": "link[href*='//cdn\\.viafoura\\.net/']",
+    "icon": "viafoura.png",
```
Comment (nurbek91) [inline src/technologies/v.json:419]: Viafoura
svg available

File: src/technologies/v.json [hunk 2]
```diff
@@ -410,6 +410,23 @@
     "scriptSrc": "\\.viabill\\.com/",
     "website": "https://viabill.com"
   },
+  "Viafoura": {
+    "cats": [
+      86
+    ],
+    "description": "Viafoura is an audience engagement and social monetization platform.",
+    "dom": "link[href*='//cdn\\.viafoura\\.net/']",
```
Comment (nurbek91) [inline src/technologies/v.json:418]: if js available no need add additional **dom** and **scriptscr** or you need examples where they are needed

File: src/technologies/v.json [hunk 3]
```diff
@@ -410,6 +410,23 @@
     "scriptSrc": "\\.viabill\\.com/",
     "website": "https://viabill.com"
   },
+  "Viafoura": {
+    "cats": [
+      86
+    ],
+    "description": "Viafoura is an audience engagement and social monetization platform.",
```
Comment (nurbek91) [inline src/technologies/v.json:417]: "description": "Viafoura is an audience engagement and social monetisation platform.",

File: src/technologies/v.json (hunk 1 above)
Comment (nurbek91) [reply src/technologies/v.json:419]: Do you see an opportunity to fragment each product separately?
Viafoura COMMUNITY ENGAGEMENT ? 86, 76
Viafoura COMMUNITY CHAT cat 52
Viafoura LIVE BLOG cat 11

image

**Review by nurbek91 — CHANGES_REQUESTED** [review]

File: src/technologies/v.json (hunk 1 above)
Comment (dhavalveera) [reply src/technologies/v.json:419]: no, I'm not able to find any trace opportunity to identify or segregate the 3

File: src/technologies/v.json (hunk 2 above)
Comment (dhavalveera) [reply src/technologies/v.json:418]: https://www.sgvtribune.com/

in this example = **link** + **scriptSrc** is being used and their `js` is quite different

File: src/technologies/v.json (hunk 1 above)
Comment (dhavalveera) [reply src/technologies/v.json:419]: what we can do on this? as we've the `link` + `scriptSrc` & `js` where I'm unable to segregate the 3 different products

File: src/technologies/v.json (hunk 2 above)
Comment (nurbek91) [reply src/technologies/v.json:418]: we can add this js dfm_viafoura_options and remove dom and scriptSrc

File: src/technologies/v.json (hunk 1 above)
Comment (nurbek91) [reply src/technologies/v.json:419]: It is necessary to look for and study current examples of how this technology is added to the site. Like a live chat or like a blog or something else, in this way we can understand what kind of product it is

Comment (nurbek91) [reply src/technologies/v.json]: If it is not possible, then this is not critical; the main thing is to correctly indicate the main category

Comment (dhavalveera) [reply src/technologies/v.json]: we've to first find the different examples such as  Chat or Blog, based on that we've to study, currently I'm not finding any scope to differentiate the Chat & Blog

**Review by nurbek91 — APPROVED** [review]

**Pull request merged**